### ISSUE TITLE
[SF_ACC][ui5-combobox]: missing tooltip and should be aria-hidden for Select Options icon

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#9600]
### Bug Description

two issue:
1. Select Options icon is missing tooltip
2. the Select Options icon should be aria-hidden for screen reader just like ui5-date-picker

### Affected Component

ui5-combobox

### Steps to Reproduce

1.
2.
3.
...

### Log Output, Stack Trace or Screenshots

<img width="294" alt="image" src="https://github.com/user-attachments/assets/7b390f85-4422-4af8-8c07-26717079c37a">

### Priority

Medium

### UI5 Web Components Version

1.24.x

### Browser

Chrome, Edge, Firefox, Safari

### Organization

SAP SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by hristop** [comment]
Hi @GongRichard ,

This request would need to go through the Inner Source process and be synchronised with the SAP Central Design team. Please contact your design colleagues and let them know that they will need to raise a BLI to the central design team. 
Once they do so, can you please pust the number of the BLI here for reference?

Best Regards,
Hristo

**Comment by GongRichard** [comment]
Hi @hristop ,
This issue was raised by SF a11y testing team and was not from design team. Why need design colleagues to raise a BLI?
Thanks,
Richard

**Comment by hristop** [comment]
Hi @GongRichard ,

As we need a new update of the design spec in order to implement the feature, as per the inner source process, someone would have to contact the Central Design team. Usually, at least I think so, this is the design responsible from the LOB side. If you can raise a design BLI you can also do so. 

Best Regards,
Hristo

**Comment by GongRichard** [comment]
Hi @hristop,
I checked with our UX here, they don't know the BLI process.
Thanks,
Richard

**Comment by GongRichard** [comment]
Hi @hristop,

Is this https://jira.tools.sap/browse/FIORITECHP1-31495 ok for you?

Thanks,
Richard

**Comment by hristop** [comment]
Hi @GongRichard ,

Yes, this is fine for us. Once the Design is ready, we will receive P1 BLI from the design team, via which we will handle this request. Please continue monitoring the progress in JIRA. 

Best Regards,
Hristo
